Mathlib/Data/Set/Card.lean

```diff
@@ -1,7 +1,7 @@
 /-
 Copyright (c) 2023 Peter Nelson. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Peter Nelson
+Authors: Peter Nelson, Yury G. Kudryashov
```

Comment:
We use `Authors` for the list of main authors of a file, not of all people who added a line or two.
```suggestion
Authors: Peter Nelson
```